networkx/algorithms/clique.py

```diff
@@ -84,17 +84,17 @@ def enumerate_all_cliques(G):
     queue = deque(([u], sorted(nbrs[u], key=index.__getitem__)) for u in G)
     # Loop invariants:
     # 1. len(base) is nondecreasing.
-    # 2. (base + cnbrs) is sorted with respect to the iteration order of G.
-    # 3. cnbrs is a set of common neighbors of nodes in base.
+    # 2. (base + cnbors) is sorted with respect to the iteration order of G.
+    # 3. cnbors is a set of common neighbors of nodes in base.
```

Comment:
I'm guessing these are abbreviations for the same thing, but I'm not terribly confident

Reply:
Yeah, this shouldn't be changed.
I mean it can be, but we just become British then.

Reply:
I should note that both `cnbrs` and `cnbors` are present... I've dropped this commit...